Xamarin.Essentials.sln

```diff
@@ -19,12 +19,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tests", "Tests\Tests.csproj
 EndProject
```

Comment:
You should probably discard the change to this file

Reply:
I will remove it, thanks 👍